scylla-cql/src/frame/response/result.rs

```diff
@@ -301,13 +301,13 @@ impl CqlValue {
 
     #[cfg(test)]
     #[cfg(feature = "chrono-04")]
-    fn as_naive_date_04(&self) -> Option<chrono_04::NaiveDate> {
+    pub(crate) fn as_naive_date_04(&self) -> Option<chrono_04::NaiveDate> {
         self.as_cql_date().and_then(|date| date.try_into().ok())
     }
 
     #[cfg(test)]
     #[cfg(feature = "time-03")]
-    fn as_date_03(&self) -> Option<time_03::Date> {
+    pub(crate) fn as_date_03(&self) -> Option<time_03::Date> {
         self.as_cql_date().and_then(|date| date.try_into().ok())
     }
 
```

Comment:
:thought_balloon: Wouldn't we like to have these functions available as `pub` out of test context, too? Such API might be useful e.g. in JS Driver.